doc/md/monitoring/07-notifications.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Notifications
```

Comment:
are we calling this "webhooks" or "notifications" or "subscriptions"? we need to be consistent in the UI, docs, and marketing

Reply:
I think it is best to go with webhooks, I will change it to be consistent